sklearn/model_selection/_search.py

```diff
@@ -532,25 +532,51 @@ def inverse_transform(self, Xt):
         self._check_is_fitted('inverse_transform')
         return self.best_estimator_.transform(Xt)
 
-    def _fit(self, X, y, groups, parameter_iterable):
-        """Actual fitting,  performing the search over parameters."""
+    @property
```

Comment:
I don't see how this helps anything and perhaps the previous version was easier to read...?